BuildRadiator/Client/Controllers/DashboardController.js

```diff
@@ -9,49 +9,93 @@
     ctrl.committerLimit = 11;
 
     function onMessageUpdate( message ) {
-      var tile = ctrl.tiles.find( function( tile ) {
+      var tiles = ctrl.tiles.filter( function( tile ) {
         return tile.type === 'message'
           && tile.config.messageKey === message.key;
       } );
 
-      if ( !tile ) {
-        return;
+      if ( tiles.length ) {
+        message.contentHtml = $sce.trustAsHtml( message.content );
       }
 
-      message.contentHtml = $sce.trustAsHtml( message.content );
-
-      tile.message = message;
+      tiles.forEach( function( t ) {
+        t.message = message;  
+      } );
     };
 
     function onProjectUpdateError( build ) {
-      var tile = ctrl.tiles.find( function( tile ) {
+      var projectTiles = ctrl.tiles.filter( function( tile ) {
```

Comment:
Updated to use `filter` in case the same build was placed multiple times on the same page. Not used in our set up but could be useful for the future.